<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallPaperClass</class>
 <widget class="QWidget" name="WallPaperClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>471</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>WallPaper</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>471</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnSetting">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>32</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="WallPaper.qrc">
       <normaloff>:/WallPaper/Resources/70BasicIcons-all-33.png</normaloff>:/WallPaper/Resources/70BasicIcons-all-33.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMinimise">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>32</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="WallPaper.qrc">
       <normaloff>:/WallPaper/Resources/zuixiaohua_1.png</normaloff>:/WallPaper/Resources/zuixiaohua_1.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>32</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="WallPaper.qrc">
       <normaloff>:/WallPaper/Resources/70BasicIcons-all-06.png</normaloff>:/WallPaper/Resources/70BasicIcons-all-06.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/WallPaper/Resources/线性关闭.png</normaloff>:/WallPaper/Resources/线性关闭.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WallPaper.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>actionExit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>WallPaperClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
